### ISSUE TITLE
Resource type terms for HydroShare terms page

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#2676]
@pkdash - we currently do not have definitions of the resource type terms on the HydroShare terms page. Per request from @dtarb, I've compiled some text that I think we should put at the top of the https://www.hydroshare.org/terms/ page. This is primarily text that came out of @dtarb 's discussion of resource types that he wrote for Olin. 

Under this issue, the header "Collection Resource Type Description" and the paragraph that follows it should be removed from the terms page since there are no definitions under that header.

@pkdash - This is just a simple update of text on  the terms page, so if someone else could handle this quickly, feel free to reassign.

Here's the text: 
[Resource_Type_Terms.txt](https://github.com/hydroshare/hydroshare/files/1804030/Resource_Type_Terms.txt)

**Comment by dtarb** [comment]
@horsburgh I am happy to have @sblack-usu handle this so @pkdash can focus on composite resources.  But please say where you want the text provided to go on the https://www.hydroshare.org/terms/ page.  The page already has descriptions of the resources under major headings on that page, just not following the terms, URI, Label, Description pattern.  Also, I think part of what was asked was terminology for the new aggregation types

**Comment by horsburgh** [comment]
@dtarb - this is fine. 

@sblack-usu - the text that is in the "Resource_Type_Terms.txt" file linked above should be posted on the http://www.hydroshare.org/terms/ page. It should go at the beginning of the page, just after the title "Description of HydroShare Resource Types and Metadata Terms".  Once you've pasted this text in there, you need to get rid of the descriptive/definition paragraph that immediately follows the title for each resource type (because the definitions are now in the text you will be adding from "Resource_Type_Terms.txt". The titles of each section also need to change. For example, the following is how the Generic Resource section should look:

### Metadata Terms for the Generic Resource Type 
To describe the entities of a Generic resource in HydroShare, we choose the standard Dublin Core metadata elements as the resource science metadata. Each of these elements are defined below. All resource types in HydroShare use the standard Dublin Core metadata elements, but some resource types use additional, resource type-specific metadata terms as defined in the following sections.

- **Term**: name
- **URI**: http://www.hydroshare.org/terms/name
- **Label**: Name
- **Description**: Name of the creator or contributor.

- . . . 

After the Generic Resource Type, the rest of the sections need only have the title and then the terms. There is no need for the text paragraph following the title in each section. Section titles should follow what I did for the generic resource above:

Metadata terms for the Time Series Resource Type
Metadata terms for the Referenced Time Series Resource Type
. . . Etc.

You can remove the section for Collection Resource Type because there are no custom metadata terms for that resource type.

All of the text for the Composite Resource Type Description can be replaced using the text I provided in Issue #2572. @dtarb - Aggregation types are defined in the text file I attached to #2572, which I am now going to assign to @sblack-usu because he can modify the all of the text on the terms page at the same time.

**Comment by sblack-usu** [comment]
@horsburgh I made the changes, though I'd appreciate it if you'd do a quick proof read of the changes I made.  I was careful to make sure I added everything you asked for, though I wouldn't be surprised if I missed something.  I'll also ping you with a link to see it running on my server.

https://github.com/hydroshare/hydroshare/commit/252d6f787614b4be35ddcec6a7f7ad4781f916bd

**Comment by sblack-usu** [comment]
Caught something i missed already.  Here is another commit.
https://github.com/hydroshare/hydroshare/commit/7590e6ded1ae367c2a259f32ebfa46b9378b98e4

**Comment by sblack-usu** [comment]
diff of both commits against develop  https://github.com/hydroshare/hydroshare/compare/develop...2676-resource-type-terms

**Comment by horsburgh** [comment]
@sblack-usu - tested on your latest deploy of the terms page. Generally, it looks good. Please see the comment I just made on #2572 - there's some changes that need to be made based on that. Also, please add the following text under the title "Metadata terms for the time series resource type":

The time series resource type is based on version 2.0 of the Observations data model. You can learn more about ODM2 at htt://www.odm2.org. Some of the metadata terms for the time series resource type are based on controlled vocabularies for ODM2 that are managed at http://vocabulary.odm2.org.

Under the "Metadata terms for the ModFlow Model Instance Resource Type, there is a line with a single "w" character that needs to be removed:

 
![image](https://user-images.githubusercontent.com/3485902/38145084-d8df4ad0-3404-11e8-99a9-dca6baa8105e.png)

**Comment by horsburgh** [comment]
@dtarb - The text in the HydroShare terms page now says that we are replacing some of the individual resource types with the composite resource. However, the composite resource isn't fully available yet. Do you want @sblack-usu to move this new terms page with the updated text to deployment?

Also - this page is going to need a pretty big overhaul in the near future if we see it as being really important. As I read down through the page, I noticed a few things:

1. Once composite resource is complete, we should just call it a resource, get rid of the references to "composite resources" and simplify the wording. The Dublin core terms are now under the Generic Resource type, but should get moved to be under a new section for standard metadata terms for all resources.
2. If we do number 1, the metadata terms that are now described under resource types would need to be instead described under content types that can reside within a resource (formerly known as composite resource).
3. Many of the terms we have designed have Terms, URIs, and Labels that do not match. At the very least, the term and the URI should match and should use a convention established for all of our metadata terms - lowercased first word, upper cased subsequent words, no spaces between words. This makes the terms URI/URL friendly. The purpose of the URI is to uniquely identify each term, so it really should match the term exactly.

I don't think Scott needs to resolve all of this now, but we should probably discuss how we want to reconcile all of this when the composite resource functionality is complete.

**Comment by dtarb** [comment]
I like the direction this is going.  I think we should move this to deployment.  I think that enough of the composite resource is available for this to be better than what is currently in terms.  

I think that some of the things @horsburgh noted can be easily addressed, and should be addressed.  Specifically:
1. The section heading "Metadata terms for the generic resource type" should be changed to "Common metadata terms across all resource types" and the lead in paragraph should be: "We chose standard Dublin Core metadata elements to describe resource science metadata.  This section defines metadata elements used in all resources.  Some resource types use additional, resource type-specific metadata terms that are defined in sections that follow this section."
2. I do not know the degree to which sequencing matters in this document, but if it does, I suggest bringing "Metadata terms for the composite resource Type" to before metadata terms for the time series resource type.
3. If number 2 is done then for the typed resources that have been implemented as composite resource aggregations we should change the corresponding heading information to refer to aggregation too, specifically 
    - "Metadata terms for the Time Series Resource type and Aggregation"
    - "Metadata terms for the Multidimensional (NetCDF) Resource type and Aggregation"
    - "Metadata terms for the Geographic Raster Resource Type and Aggregation"

While examining this I note that we are missing completely a definition of terms for the Geographic Feature Resource Type.  Issue #2057 is about this, assigned to @zhiyuli.  I suggest when it comes to code review asking @zhiyuli to do the review, so he can be reminded to work on this issue.

4. I also noticed that there are spaces (extra lines) in some of the html formatting that would be good to improve.

**Comment by horsburgh** [comment]
@dtarb - I think the extra lines were there to indicate groupings of terms, but I'm not sure they are used consistently so can probably be removed.
